src/core/utils/EventNormalizer.ts

```diff
@@ -925,6 +929,13 @@ export class EventNormalizer {
      * @param {Event} ev
      */
     _onSelectionChange(): void {
+        if (this._rangeHasChanged) {
+            // This should disappear once the "diff" system is introduced.
+            // Right now it is required so as to avoid an infinite loop when
+            // selectAll triggers a new range set... and the selection changes
+            // every time.
```

Comment:
I don't understand what the "diff" system will change to this behavior ?

Reply:
According to Christophe, the "diff" system will recognize that the new range is not different from the old range and it won't trigger a new rendering.